client/features/sharedComponents/MediaTileCarousel.css

```diff
@@ -0,0 +1,288 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); */
+
+.mtc-carousel {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: column;
+  font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 400;
+  padding: .5em;
+  overflow: hidden;
+  width: 100%;
+  margin-bottom: .5em;
+}
+
+.mtc-component-title {
+  display: flex;
+  user-select: none;
+  justify-self: left;
+}
+
+.mtc-viewport {
+  display: flex;
+  box-sizing: content-box;
+  min-width: 100%;
+  max-width: 100%;
+  margin: 8px 0;
```

Comment:
I'm seeing a lot of px in here. Is this something we can change to be relative? Just one opinion here, others might agree with using the px.

Reply:
This is because the component has to control scrolling, which is calculated in pixels on the browser.